app/assets/onepager/js/send.js

```diff
@@ -76,13 +76,74 @@ $(document).ready(function() {
   $('#advanced_toggle').on('click', function() {
     advancedToggle();
   });
+  $('.cb-value').click(function() {
+    var mainParent = $(this).parent('.toggle-btn');
+
+    if ($(mainParent).find('input.cb-value').is(':checked')) {
+
+      var username = $('.username-search').select2('data')[0] ? $('.username-search').select2('data')[0].text : '';
+      const url = '/tip/address/' + username;
+
+      fetch(url, {method: 'GET', credentials: 'include'}).then(function(response) {
+        return response.json();
+      }).then(function(json) {
+        if (json.addresses.length > 0 && json.addresses[0] !== '0x0') {
+          $('#send-stream-invite').css('display', 'none');
+          var validToken = validateStreamingToken();
+
+          if (!validToken) {
+            $(mainParent).find('input.cb-value').prop('checked', false);
+            _alert('The token you selected is not valid for streaming', 'error');
+          } else {
+            var proxyAddress = sablier_proxy_address();
+
+            if (!proxyAddress.startsWith('0x')) {
+              $(mainParent).find('input.cb-value').prop('checked', false);
+              _alert('Streaming tips is not available on this network', 'error');
+            } else {
+              $(mainParent).addClass('active');
+              $('#sablier-opts').css('display', 'block');
+            }
+          }
+        } else {
+          _alert({ message: gettext('Recipient does not have any payment address set. Click in the button below if you want to send an invite') }, 'warning');
+          $('#send-stream-invite').css('display', 'inline-block');
+        }
+      });
+    } else {
+      $(mainParent).removeClass('active');
+      $('#sablier-opts').css('display', 'none');
+    }
+  });
+  $('#duration').durationPicker({
+    lang: 'en',
+    onChanged: function(time) {
+      var tokenAddress = $('#token').val();
+
+      if (tokenAddress && tokenAddress != '0x0') {
+        var token = tokenAddressToDetails(tokenAddress);
+        var amount = parseFloat($('#amount').val());
+        var streamTime = parseInt(time);
+
+        if (amount > 0 && streamTime > 0) {
+          var deposit = getEffectiveStreamDeposit(amount, token.decimals, streamTime);
+          var streamPerSec = deposit.div(web3.toBigNumber(time)).div(web3.toWei(web3.toBigNumber(1), 'ether')).toNumber();
+          var text = `It will stream ${streamPerSec >= 0.001 ? `~${streamPerSec.toFixed(3)}` : '<0.001'} ${token.name} / sec`;
+
+          $('#stream-per-sec').html(text);
+        } else {
+          $('#stream-per-sec').html('</br>');
+        }
+      }
+    },
+    showSeconds: false
+  });
   $('#amount').on('keyup blur change', updateEstimate);
   $('#token').on('change', updateEstimate);
   $('#send').on('click', function(e) {
     e.preventDefault();
     if ($(this).hasClass('disabled'))
       return;
-    loading_button($(this));
```

Comment:
did u mean to remove this?

Reply:
It was below the code I added. Just restored it to the right place